noise.cpp

```diff
@@ -124,14 +124,20 @@ double PerlinNoise::interpolate(double a, double b, double w) {
 	return (b - a) * (3.0 - w * 2.0) * w * w + a;
 }
 
-double Noise::wiggle(double base, double min, double max, double step) {
-	bool up = random() < 0.5;
+// rand_exp: Bias for the random number generator. >1 outputs higher numbers on average, <1 outputs lower numbers on average
+// distance_exp: Bias for the distance from the center. >1 increases the percieved distance, <1 decreases the percieved distance
+double Noise::wiggle(double base, double center, double min, double max, double step, double rand_exp, double distance_exp) {
```

Comment:
Since this function is so highly specialized now, I don't think it belongs in Noise anymore, it probably should go in SpriteWiggler since that's what it has been redesigned for.